src/elements/hx-busy/index.spec.js

```diff
@@ -6,31 +6,81 @@ import { fixture, expect } from '@open-wc/testing';
     const template = '<hx-busy>';
 
-    it('should be instantiated with hx-defined attribute', async () => {
-        const el = /** @type {HXBusyElement} */ await fixture(template);
+    describe('instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            const attr = el.hasAttribute('hx-defined');
 
-        expect(el.getAttribute('hx-defined')).to.exist;
-    });
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            const prop = el.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered light DOM should NOT equal simple template ${template}`, async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
 
-    it('should have aria-hidden attribute set to true', async () => {
-        const el = /** @type {HXBusyElement} */ await fixture(template);
+            expect(el).lightDom.to.not.equal(template);
+        });
 
-        expect(el.getAttribute('aria-hidden')).to.be.equal(String(true));
+        it(`should NOT have a Shadow DOM`, async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            const sd = el.shadowRoot;
+
+            expect(sd).to.be.null;
+        });
     });
 
-    it(`the rendered light DOM should NOT equal simple template ${template}`, async () => {
-        const el = /** @type {HXBusyElement} */ await fixture(template);
+    describe('test $onConnect method', () => {
+        it('should not be paused on connect', async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            const prop = el.hasAttribute('paused');
+
+            expect(prop).to.be.false;
+        });
+
+        it('should have aria-hidden attribute', async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            const attr = el.hasAttribute('aria-hidden');
+            expect(attr).to.be.true;
+        });
 
-        expect(el).lightDom.to.not.equal(template);
+        it('should have aria-hidden attribute set to true', async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            const attr = el.getAttribute('aria-hidden');
+
+            expect(attr).to.be.equal(String(true));
+        });
     });
 
-    it(`should NOT have a Shadow DOM`, async () => {
-        const el = /** @type {HXBusyElement} */ await fixture(template);
-        const sd = el.shadowRoot;
+    describe('test <hx-busy> methods', () => {
+        it('should be able to pause element', async () => {
+            const el = /** @type {HXBusyElement} */ await fixture(template);
+            el.paused = true;
+            const attr = el.hasAttribute('paused');
```

Comment:
@lalithkarikelli @BY00565233 

Please notice I am assigning variables, then testing the value in the `expect` statements.